### DISCUSSION TITLE
Standalone executable Pluto notebook

### DISCUSSION THREAD

**Discussion by Oblynx** [fonsp/Pluto.jl#2339] [Pluto development]
I want to create a GUI for a bluetooth library in a Pluto notebook. The target audience needs to run the notebook server locally on their laptops in order to communicate with Bluetooth. However, they are a non-technical audience and should be able to run an executable without Julia installed.

Do we have any experience compiling the Pluto server + a notebook into an "app" with PackageCompiler?
I tried a straightforward compilation, but it didn't work. Glad for any hints or experiences!

Should I also ask on Discourse?

**Comment by pankgeorg** [comment]
Γεια σου Κωνσταντίνε! Check if this is closer to what you need - https://github.com/JuliaPluto/PlutoDesktop/
